airflow_aws_executors/batch_executor.py

```diff
@@ -99,6 +99,27 @@ def sync(self):
                 task_key = self.active_workers.pop_by_id(job.job_id)
                 self.success(task_key)
 
+    def trigger_tasks(self, open_slots: int) -> None:
```

Comment:
This should be fixed upstream to base Airflow. The major reasoning is that `order_queued_tasks_by_priority` is an Airflow 2.0 method. So this introduction would break backwards compatibility with Airflow 1.10.14 and lower.